src/test/groovy/graphql/ScalarsBooleanTest.groovy

```diff
@@ -55,19 +55,19 @@ class ScalarsBooleanTest extends Specification {
         Scalars.GraphQLBoolean.getCoercing().serialize(value, GraphQLContext.default, Locale.default) == result
 
         where:
-        value                        | result
-        true                         | true
-        "false"                      | false
-        "true"                       | true
-        "True"                       | true
-        0                            | false
-        1                            | true
-        -1                           | true
-        new Long(42345784398534785l) | true
```

Comment:
A bunch of constructors were deprecated in Java 9. Now we're on Java 11 let's update these